components/HackneyDS/styles/main.scss

```diff
@@ -0,0 +1,5679 @@
+@import url(https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap);
```

Comment:
This file has over 5000 lines of css. Can you please separate the relevant styles and put them in separate files as we currently do

Reply:
discussed in slack, thx